sync/notify/msg.py

```diff
@@ -52,7 +52,8 @@ def status_str(result,  # type: Union[Result, SubtestResult, TestResult]
 
     if include_other_browser:
         other_browser_values = []
-        for other_browser, job_results in iteritems(result.statuses):
+        ordered_statuses = OrderedDict(sorted(result.statuses.items()))
```

Comment:
You don't actually need an `OrderedDict` here, right? You can just do `for other_browser, job_results in sorted(result.statuses.items())`

Reply:
Oh, you are right. Fixed!